e2e/fake-timers/clear-real-timers/__tests__/clearRealTimers.test.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+test('allows clearing not faked timers', () => {
+  const timer = setTimeout(() => {
+    throw new Error('Should not throw');
+  }, 1000);
+
+  jest.useFakeTimers();
+
+  clearTimeout(timer);
+});
```

Comment:
@SimenB This tests now throws an error instead of `console.log()`. Now if I was remove `shouldClearNativeTimers: true`, which is hardcoded in `modernFakeTimers.ts`, and run `fakeTimers.test.ts`:

<img width="585" alt="Screenshot 2022-03-28 at 08 47 00" src="https://user-images.githubusercontent.com/72159681/160334709-d6d8b10f-8c7d-414b-89b0-accf2931e344.png">

Adding back `shouldClearNativeTimers: true` makes the test green again. The idea here is to test if the hardcoded `shouldClearNativeTimers: true` does the job. As described in https://github.com/sinonjs/fake-timers/issues/406, if a user is calling `clearTimeout` the timeout gets cleared.

Or I still miss something?

Reply:
that seems good! 👍